scripts/run-observer.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+SHARD=metachain
+cd ..
+
+# build docker image
+IMAGE_NAME=elrond-node-image-test
+docker image build . -t ${IMAGE_NAME} -f ./docker/Dockerfile
+
+# generate a new BLS key
+OUTPUT_FOLDER=~/output/keys
+if [ ! -f "${OUTPUT_FOLDER}/validatorKey.pem" ]; then
+  mkdir -p ${OUTPUT_FOLDER}
+  docker run --rm --mount type=bind,source=${OUTPUT_FOLDER},destination=/keys --workdir /keys elrondnetwork/elrond-go-keygenerator:latest
+fi
+
+## run docker image
+PORT=8080
+docker run -d -p 8080:${PORT} --mount type=bind,source=${OUTPUT_FOLDER}/,destination=/data ${IMAGE_NAME} --validator-key-pem-file="/data/validatorKey.pem" --log-level *:DEBUG --log-save --destination-shard-as-observer=${SHARD}
+
+export ADDRESS_WITH_ROUTE=http://localhost:${PORT}/network/status
+
+#check if 'curl' is installed
+if ! command -v curl &> /dev/null; then
+    apt install -y curl
+fi
+
+#check if 'jq' is installed
+if ! command -v jq &> /dev/null; then
+    apt install -y jq
+fi
+
+
+check_nonce_grows() {
+  echo "node is running...wait for the tree to be synced"
```

Comment:
Pleas move this on L37 as to align with the rest of the repos

Reply:
done.